src/Microsoft.Health.Dicom.Core/Features/Store/StoreValidatorResult.cs

```diff
@@ -0,0 +1,33 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Dicom.Core.Features.Store;
+
+public sealed class StoreValidatorResult
```

Comment:
Perhaps `StoreValidationResult` and `StoreValidationResultBuilder`

Reply:
Done.